src/about.cpp

```diff
@@ -2,20 +2,18 @@
 #include "ui_about.h"
 
 #define _STR(X) #X
-#define STR(X)  _STR(X)
+#define STR(X) _STR((X))
```

Comment:
This add a parentheses around the value to transform in string.
This should work better :
```
#define _STR(...) # __VA_ARGS__
#define STR(X) _STR(X)
```

Reply:
Fixed in last forced push.